ffi/src/log.rs

```diff
@@ -0,0 +1,33 @@
+static LOG_INITED: std::sync::atomic::AtomicBool = std::sync::atomic::AtomicBool::new(false);
+
+#[diplomat::bridge]
+pub mod ffi {
+    use std::{fs::File, path::PathBuf};
+
+    use super::LOG_INITED;
+    use std::fs;
+
+    #[diplomat::opaque]
+    pub struct Log;
+
+    impl Log {
+        pub fn init_log_with_env() {
+            if LOG_INITED.load(std::sync::atomic::Ordering::Relaxed) {
+                return;
+            }
+            LOG_INITED.store(true, std::sync::atomic::Ordering::Relaxed);
+
+            let filepath: PathBuf = std::env::var("IRONRDP_LOG_FILE")
```

Comment:
suggestion: Use `IRONRDP_LOG`, like in `ironrdp-client`

Reply:
updated